src/hardware/serialport/directserial.h

```diff
@@ -32,38 +32,41 @@
 
 class CDirectSerial : public CSerial {
 public:
-	CDirectSerial(Bitu id, CommandLine* cmd);
+	CDirectSerial(const CDirectSerial &) = delete; // prevent copying
```

Comment:
Change here only replaces the `Bitu` member counters with `uint32`s.  These are not used to size STL containers.